MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfMapsPolygonStroke                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of polygon layer stroke.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfMapsPolyLineStroke                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of polyline layer in maps.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfMapsTooltipTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Font size of tooltip text in maps.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfMapsDataLabelTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Font size on data label in maps.      
+            <br/>
+            <br/>
+        </td>
+    </tr>
+</table>
+
+
+## SfSchedule
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+
+    <tr>
+        <td rowspan="30">
+            SfSchedulerStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfSchedulerTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfScheduler without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfSchedulerTheme">CommonTheme</x:String> 
+                <Color x:Key="SfSchedulerNormalHeaderTextColor">Red</Color>
+                <Color x:Key="SfSchedulerNormalViewHeaderDayTextColor">Blue</Color>
+                <Color x:Key="SfSchedulerNormalViewHeaderDateTextColor">Blue</Color> 
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfSchedulerHoverColor    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of scheduler when it is hovered.
```

Comment:
Color of scheduler cells when it is hovered

Reply:
Updated